osu.Game.Rulesets.Osu/Mods/OsuModSuddenDeath.cs

```diff
@@ -13,5 +18,11 @@ public class OsuModSuddenDeath : ModSuddenDeath
         {
             typeof(OsuModTargetPractice),
         }).ToArray();
+
+        [SettingSource("Fail when missing on a slider tail")]
+        public BindableBool SliderTailMiss { get; } = new BindableBool();
+
+        protected override bool FailCondition(HealthProcessor healthProcessor, JudgementResult result) =>
+            (SliderTailMiss.Value && result.HitObject is SliderTailCircle && result.Type == HitResult.IgnoreMiss) || base.FailCondition(healthProcessor, result);
```

Comment:
The two return statements in the old diff were more readable to me, but wait for other opinions.

Reply:
Agree, not sure what I'm reading here.

Reply:
Fixed in https://github.com/ppy/osu/pull/31232/commits/fd1cc34e3fd01747eb132c04b831a22429be7c99